.goreleaser.yaml

```
@@ -0,0 +1,63 @@
+version: 1
+
+project_name: lava
+
+gomod:
+  proxy: true
+
+builds:
+  - id: lava
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
```

For now I would remove `windows`. It is not officially supported.

```suggestion
    goos:
      - linux
      - darwin
```